script.js

```diff
@@ -0,0 +1,181 @@
+
+for (let i = 0; i < 3; i += 1) {
+  const speakerContainer = document.querySelector('.speakers');
+  const speakerList = document.createElement('li');
+  const speakerImg = document.createElement('img');
+  const speakerInfo = document.createElement('div');
+  const speakerName = document.createElement('h4');
+  const speakerPosition = document.createElement('p');
+  const speakerStory = document.createElement('p');
+  speakerList.classList.add('speaker-list');
+  speakerImg.classList.add('speaker-img');
+  speakerInfo.classList.add('speaker-info');
+  speakerName.classList.add('speaker-name');
+  speakerStory.classList.add('speaker-story');
+  speakerPosition.classList.add('speaker-position');
+  speakerStory.classList.add('speaker-story');
+  speakerInfo.appendChild(speakerName);
+  speakerInfo.appendChild(speakerPosition);
+  speakerInfo.appendChild(speakerStory);
+  speakerList.appendChild(speakerImg);
+  speakerList.appendChild(speakerInfo);
+  speakerImg.setAttribute('src', speaker[i].image);
+  speakerImg.setAttribute('alt', speaker[i].imageText);
+  speakerName.textContent = speaker[i].name;
+  speakerPosition.textContent = speaker[i].title;
+  speakerStory.textContent = speaker[i].info;
+  speakerContainerFirstHalf.appendChild(speakerList);
+  speakerUlistContainer.appendChild(speakerContainerFirstHalf);
+  speakerContainer.appendChild(speakerUlistContainer);
+  speakerContainer.appendChild(moreBtn);
+}
+
+for (let i = 3; i < 6; i += 1) {
+  const speakerContainer = document.querySelector('.speakers');
+  const speakerList = document.createElement('div');
+  const speakerImg = document.createElement('img');
+  const speakerInfo = document.createElement('div');
+  const speakerName = document.createElement('h4');
+  const speakerPosition = document.createElement('p');
+  const speakerStory = document.createElement('p');
+  speakerList.classList.add('speaker-list');
+  speakerImg.classList.add('speaker-img');
+  speakerInfo.classList.add('speaker-info');
+  speakerName.classList.add('speaker-name');
+  speakerStory.classList.add('speaker-story');
+  speakerPosition.classList.add('speaker-position');
+  speakerStory.classList.add('speaker-story');
+  speakerInfo.appendChild(speakerName);
+  speakerInfo.appendChild(speakerPosition);
+  speakerInfo.appendChild(speakerStory);
+  speakerList.appendChild(speakerImg);
+  speakerList.appendChild(speakerInfo);
+  speakerImg.setAttribute('src', speaker[i].image);
+  speakerImg.setAttribute('alt', speaker[i].imageText);
+  speakerName.textContent = speaker[i].name;
+  speakerPosition.textContent = speaker[i].title;
+  speakerStory.textContent = speaker[i].info;
+  speakerContainerSecondHalf.appendChild(speakerList);
+  speakerUlistContainer.appendChild(speakerContainerSecondHalf);
+  speakerContainer.appendChild(speakerUlistContainer);
+  speakerContainer.appendChild(moreBtn);
+}
+
+const menuBtn = document.querySelector('.open-mobile-menu');
```

Comment:
**[OPIONAL]** - Also, for this functionality, Please look into how to use the [slice](https://www.w3schools.com/jsref/jsref_slice_array.asp) JavaScript array method to slice a specific section of an array. this will help make your code look cleaner, more maintainable, and avoid repetitions